### PR TITLE
Fixed broken link, replaces file links with permalinks

### LINKED ISSUE #9330: Broken link to code path in documentation (dynamic imports)
## Description
The link to a code sample file in the documentation on **Dynamic Imports** has changed in the repository.

## Steps to reproduce
[Dynamic imports](https://docs.dev.to/frontend/dynamic-imports/). the codeEditor.jsx#L11 file no longer exists.
https://github.com/forem/forem/blob/master/app/javascript/chat/codeEditor.jsx#L11

second link in the examples of dynamic import usage on DEV.

### PR DESCRIPTION
**Pull request by oyeyipowale**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
Broken file link in the **Dynamic Imports** documentation section. Link replacement with permalink.

## Related Tickets & Documents
Closes #9330 

## QA Instructions, Screenshots, Recordings

Go to [Dynamic imports doc](https://docs.dev.to/frontend/dynamic-imports/) and click on the two examples link.

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: docs/frontend/dynamic-imports.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ Explorer which are both unsupported browsers for DEV. They allow you to import
...
-JavaScript module on the fly for functionality that is not needed immediately
+Should you use them everywhere? No. They are a great tool when you need to load
+a JavaScript module on the fly for functionality that is not needed immediately
 for the page to be usable.
 
 Here are a couple of examples of dynamic import usage on DEV:
 
 - The
-  [Onboarding flow](https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/packs/Onboarding.jsx#L28).
+  [Onboarding flow](https://github.com/forem/forem/blob/0633d85b6b0e083bb7b21b11642b2b17d3fe9de6/app/javascript/packs/Onboarding.jsx#L21).
```
Comment (jacobherrington) [inline docs/frontend/dynamic-imports.md:21]:
```suggestion
  [Onboarding flow](https://github.com/forem/forem/blob/master/app/javascript/packs/Onboarding.jsx#L21).
```

File: docs/frontend/dynamic-imports.md [hunk 2]
```diff
@@ -11,17 +11,17 @@ Explorer which are both unsupported browsers for DEV. They allow you to import
...
 - In
-  [connect](https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/chat/codeEditor.jsx#L11)
-  (chat)
+  [homepage](https://github.com/forem/forem/blob/0633d85b6b0e083bb7b21b11642b2b17d3fe9de6/app/javascript/packs/homePage.jsx#L59)
```
Comment (jacobherrington) [inline docs/frontend/dynamic-imports.md:23]:
```suggestion
  [homepage](https://github.com/forem/forem/blob/master/app/javascript/packs/homePage.jsx#L59)
```

**Comment by rhymes** [comment]
@jacobherrington we can't use the branch explicitly because lines get moved around, we need to use permalinks to fix the example in time, see https://github.com/forem/forem/issues/9330#issuecomment-658764171

**Comment by oyeyipowale** [comment]
Thanks for being awesome 🙌

**Pull request merged**